docs/howto/configure/advanced/iframe.md

```diff
@@ -0,0 +1,242 @@
+```
+
+The JupyterLab extension is created! The following command checks if it is correctly
+loaded in the environment:
+
+```bash
+jupyter labextension list
+```
+
+Now, create the JupyterLite extension. Install jupyterlite:
+
+```bash
+pip install jupyterlite
+```
+
+Move to a directory where the extension should be tested and run the build of this
+extension for JupyterLite:
+
+```bash
+mkdir examples
+cd examples
+jupyter lite build --output-dir lite
+```
+
+The following lines show that the extension has been correctly built:
+
+```
+...
+federated_extensions:copy:ext:jupyterlab-iframe-bridge-example
+.  pre_build:federated_extensions:copy:ext:jupyterlab-iframe-bridge-example
+...
+```
+
+A `jupyterlab-iframe-bridge-example/examples/lite/` directory containing everything
+needed for JupyterLite to work is created (notice the presence of our extension in the
+`extensions` subdirectory).
+
+## Test the extension
+
+To test the communication between a host page and an IFrame containing JupyterLite,
+create a file `jupyterlab-iframe-bridge-example/examples/index.html`. Edit this file
+with the following code:
+
+```html
+<html>
+  <title>Example bridge between a host app and JupyterLite</title>
+  <body>
+    <script type="text/javascript">
+      /* Outgoing messages */
+      function toggle() {
+        window.frames.jupyterlab.postMessage({ type: 'from-host-to-iframe' });
+      }
+
+      /* Incoming messages */
+      window.addEventListener('message', (event) => {
+        if (event.data.type === 'from-iframe-to-host') {
+          document.getElementById('chosenTheme').innerText = event.data.theme;
+        }
+      });
+    </script>
+    <h2>Below is a JupyterLite site running in an IFrame</h2>
+    <p>
+      Click the following button sends a message to the JupyterLab IFrame to toggle the
+      theme.
+    </p>
+    <p>The IFrame indicates that the current theme is: <em id="chosenTheme"></em></p>
+    <input type="button" value="Toggle the JupyterLab Theme" onclick="toggle()" />
+    <iframe
+      name="jupyterlab"
+      src="lite/"
+      width="100%"
+      height="600px"
+      sandbox="allow-scripts allow-same-origin"
+    ></iframe>
+  </body>
+</html>
+```
+
+When the user clicks on the button, the `toggle` function sends a message to the IFrame
+via the `postMessage` method. This message is intercepted by our extension which changes
+the theme. Moreover, when the host page receives a message from the IFrame notifying of
+an effective theme change, it displays it to the user.
+
+In order to visualize this process, launch a minimalist server from the `examples`
+directory:
+
+```bash
+cd examples
+python -m http.server -b 127.0.0.1
+```
+
+In a browser, at the address `http://127.0.0.1:8000`, you should be able to notice the
+communication between the host page and the IFrame (refresh the browser if necessary):
+
+https://user-images.githubusercontent.com/44410933/218273379-07d129e3-27c1-4c30-a89b-6d1a7bb61b26.mp4
```

Comment:
Maybe we can convert this video to a gif so it renders on ReadTheDocs:

https://jupyterlite--980.org.readthedocs.build/en/980/howto/configure/advanced/iframe.html